client/utils/types.ts

```diff
@@ -1,44 +1,34 @@
 import { Moment } from 'moment-timezone'
 
-export type Id = number
+type URL = string
 
-export interface Club {
+export interface Resource {
   id: Id
   name: string
   description: string
-  size: number
+  location: string
   links: {
-    bannerImage: string
-    iconImage: string
-    facebook?: string
-    twitter?: string
-    instagram?: string
-    website?: string
+    bannerImage: URL
+    iconImage: URL
   }
-  tags: Array<string>
-  events: Array<BasicEvent>
+  gallery: Array<URL>
+  overallRating: number
+  review: Array<{
+    comment: string
+    rating: number
+  }>
 }
 
-export interface BasicClub {
-  name: string
-  iconURL: string
-}
+export interface House extends Resource {}
 
-export interface Event {
-  id: Id
-  name: string
-  club: BasicClub
-  description: string
-  bannerImageURL: string
-  startTime: Moment
-  endTime: Moment
-  tags: Array<string>
-}
+export interface StudySpot extends Resource {}
 
-export interface BasicEvent {
-  name: string
-  startTime: Moment
-  endTime: Moment
-  bannerImageURL?: string
-  tags: Array<string>
+export interface Washroom extends Resource {}
+
+export type Id = number
+
+export interface Review {
+  avatarImage: string
```

Comment:
this more realistically would be a userID, I guess this means we'll have users?

Reply:
Hmm, depending on how much dev resources we have left...maybe we should just make all comments anonymous for v2?